src/api/app/views/webui/obs_factory/staging_projects/_checks.html.haml

```diff
@@ -0,0 +1,13 @@
+- if @build_id.present?
```

Comment:
Hm why is this dependent on `build_id`? Isn't there always a `build_id`?

Reply:
Not if you haven't configured the images repository or no build finished yet or publish is disabled (not sure about the last).